semcore/data-table/src/DataTable.tsx

```diff
@@ -234,12 +236,14 @@ class RootDefinitionTable extends Component<AsProps> {
       }
 
       const column = this.columns.find((column) => column.name === name);
-      columnsChildren.push({
+
+      const columnChildren = {
         get width() {
           return this.props.ref.current?.getBoundingClientRect().width || 0;
         },
         name,
-        cssVar: createCssVarForWidth(name),
+        varWidth: !varWidth || varWidth === 'inhered' ? createCssVarForWidth(name) : varWidth,
```

Comment:
What is `inhered`? May be you meant `inherited` or `inherit`?

Reply:
Yes, of course. I had a mind `inherit`